atomic_reactor/constants.py

```diff
@@ -60,3 +60,14 @@
 )
 
 DEFAULT_DOWNLOAD_BLOCK_SIZE = 10 * 1024 * 1024 # 10Mb
+
+LABEL_ALIASES = {
+    "name" : ('name', 'Name'),
```

Comment:
well this is symbolic name :) but how else I should name the name ;) I can change it so something like : alias_name?